source/javascripts/components/_Meta.js.erb

```diff
@@ -0,0 +1,65 @@
+(function() {
+
+"use strict";
+
+angular.module("BitriseWorkflowEditor").factory("Meta", function() {
+
+	var Meta = function(userMetaConfig, defaultMetaConfig) {
+		this.userMetaConfig = userMetaConfig;
+		if (!this.userMetaConfig) {
+			this.userMetaConfig = {};
+		}
+
+		this.defaultMetaConfig = defaultMetaConfig;
+	};
+
+	Meta.prototype.valueGetterSetter = function(value, key, bundleID) {
```

Comment:
Let's move the `value` to the last position, it's more convenient to use it in getter mode.